bundles/net.tourbook/src/net/tourbook/tag/Dialog_TourTag.java

```diff
@@ -36,22 +49,39 @@
  */
 public class Dialog_TourTag extends TitleAreaDialog {
```

Comment:
I opened this dialog with 4 different tags but all images are not disposed. It is very important that images are disposed when not needed anymore.

I strongly recommend to use the Sleak view to verify the image resources !

![image](https://user-images.githubusercontent.com/1283445/218257769-06fbe292-1ac2-4e1d-9cb2-ccdb7512fff9.png)


I stop for now the review until this is fixed. In the following graph you can see the files which I've not yet reviewed

![image](https://user-images.githubusercontent.com/1283445/218257897-12614fc0-f779-437b-a81f-1ce0356224d2.png)

Reply:
> I stop for now the review until this is fixed. In the following graph you can see the files which I've not yet reviewed
> 

That sounds good. I am putting the PR in draft mode until I fix all your suggestions/reviews and it is ready to be reviewed again

Reply:
I already use Sleak but it has never shown any images non disposed when I developed this feature

I used [your documentation ](https://mytourbook.sourceforge.io/mytourbook/index.php/development/documentation-dev-tips/dev-tips-sleak)but maybe I am missing something in my Eclipse configuration ?

![image](https://user-images.githubusercontent.com/25235665/218578597-769dc402-04fc-4138-b28a-d6d94e32f55c.png)

Reply:
Ok, thanks for the detailed steps, I think I was forgetting to do that :

Press `Snap & Diff` button in Sleak view, important, press it **BEFORE** opening the pref dialog

I was able to see the leaks and they should be fixed now

Reply:
> > I stop for now the review until this is fixed. In the following graph you can see the files which I've not yet reviewed
> 
> That sounds good. I am putting the PR in draft mode until I fix all your suggestions/reviews and it is ready to be reviewed again

I have set the PR back to "ready for review" as I have addressed all your code review coments